app/src/ui/diff/side-by-side-diff-row.tsx

```diff
@@ -384,18 +409,20 @@ export class SideBySideDiffRow extends React.Component<
   /**
    * Renders the line number box.
    *
+   * @param divWidth The width the line number component should be.
    * @param lineNumbers Array with line numbers to display.
    * @param isSelected  Whether the line has been selected.
    *                    If undefined is passed, the line is treated
    *                    as non-selectable.
    */
   private renderLineNumbers(
     lineNumbers: Array<number | undefined>,
+    divWidth: String,
```

Comment:
I am not sure this actually needs passed in from the render function since it is on the `props`. I think you should be able to access it direction in this method with `this.props.lineNumberWidth`.